src/Admin/Extensions/Registry.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Registry of WPGraphQL Extensions
+ *
+ * @todo This will eventually be replaced with a server registry.
+ *
+ * @package WPGraphQL\Admin\Extensions
+ * @since @todo
+ */
+
+namespace WPGraphQL\Admin\Extensions;
+
+/**
+ * Class Registry
+ *
+ * @phpstan-type Extension array{
+ *  name: non-empty-string,
+ *  description: non-empty-string,
+ *  plugin_url: non-empty-string,
+ *  support_url: non-empty-string,
+ *  documentation_url: non-empty-string,
+ *  repo_url?: string,
```

Comment:
These were in the documentation, but not used in the JSON schema.

Single source of truth means removing/updating them here is all we need for PHPStan to make sure that they're correctly consumed everywhere else.

Reply:
Ah yes. Good point. I did make a decision on the fly to leave them out for now as the UI doesn't use them. Figured we could make a decision on these fields later.